libs/ngworker/lumberjack/src/lib/errors/lumberjack-http-driver.error.ts

```diff
@@ -0,0 +1,18 @@
+export class LumberjackHttpDriverError extends Error {
```

Comment:
question: this is the repeated code you mentioned?

Reply:
No, I just added this one. All `http-driver` files have been copied into the main `src/lib` sub-directories.